emailConfig.json

```diff
@@ -0,0 +1,4 @@
+{
+    "username": "bobross69pogchamp@gmail.com",
+    "password": "ctc12345"
```

Comment:
🚫 **[detect-secrets]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
* Secret Keyword

Reply:
It is possible to move these credentials to a hidden file such as the .env file since we don't want information like that to be put onto the repo in the future, especially once we give project control to the NPO